src/common/network/NetworkUtils.cpp

```diff
@@ -135,9 +178,15 @@ uint16_t NetworkUtils::getAvailablePort() {
 
     std::unordered_set<uint16_t> portsInUse = getPortsInUse();
     uint16_t port = 0;
-    do {
-        port = folly::Random::rand32(low, static_cast<int32_t>(high) + 1);
-    } while (portsInUse.find(port) != portsInUse.end());
+    while (true) {
+        port = folly::Random::rand32(1025, low);
```

Comment:
Need some comments. Why to select a port from range [1025, low]?

Reply:
Yeah, it should have been.